plotly/session.py

```diff
@@ -76,9 +76,10 @@ def sign_in(username, api_key, **kwargs):
     # raise error if key isn't valid anywhere
     for key in kwargs:
         if key not in CREDENTIALS_KEYS and key not in CONFIG_KEYS:
-            raise exceptions.PlotlyError(
-                "{} is not a valid config or credentials key".format(key)
-            )
+            raise exceptions.PlotlyError("{} is not a valid config or "
+                                         "credentials key\n"
+                                         "Type `help(sign_in)` for a full list "
+                                         "of sign in related keys".format(key))
```

Comment:
It might be helpful for people to see these keys immediately when they get the error, so like:

```
"domain" is not a valid config or credentials key.
Valid keys include: username, api_key, stream_ids, proxy_username, proxy_password, plotly_domain, plotly_streaming_domain, plotly_api_domain, plotly_ssl_verification, plotly_proxy_authorization, world_readable.
Call help(sign_in) for more information.
```